packages/extension/src/background/index.ts

```diff
@@ -131,6 +131,25 @@ async function handleMessages(message, sender: Runtime.MessageSender) {
       },
     });
   }
+
+  if (message.type === 'REQUEST_UPDATE') {
+    // check if requestUpdateCheck is available
+    // @ts-expect-error Property 'requestUpdateCheck' does not exist on type 'Static'
```

Comment:
Had to add this ts override because `webextension-polyfill-ts` is so outdated that it does not have the correct types. I think it is fine for now because I do not want to update packages in this PR and also there is a #733

Reply:
Yes, it is really a pain to keep this outdated package. Hopefully, we can focus soon on migrating the Manifest since FF has started receiving entries now iirc.

Reply:
Solve with #1613

Reply:
So we can change this right?

Reply:
It should be solved with https://github.com/dailydotdev/apps/pull/733, #1613 only added `message.type`. Unresolving this for reference.